### ISSUE TITLE
ME Storage Bus does not show items in nested networks (Ars Nouveau + Functional Storage)

### ISSUE THREAD

**Issue by NielsPilgaard** [AppliedEnergistics/Applied-Energistics-2#8127]
Relay of https://github.com/EnigmaticaModpacks/Enigmatica9/issues/998

### Describe the bug

The Storage Bus is unable to see items in nested item networks. The setup is this:
- Storage Bus on Ars Nouveau Storage Lectern
- Storage Lectern has a Functional Storage Storage Controller connected to it

### How to reproduce the bug

- Place Storage Bus on Ars Nouveau Storage Lectern
- Storage Lectern has a Functional Storage Storage Controller connected to it

@Gregie screenshots of your setup would be helpful here 😊

### Expected behavior

For all items in connected storages to appear and be accessible

### Which minecraft version are you using?

1.20.1

### On which mod loaders does it happen?

Forge

### Crash log

https://gist.github.com/Gregie/bea8b08cdb3ee109d4c61aeb459ddeb7

**Comment by Gregie** [comment]
Screenshots of setup:
![2024-08-09_18 23 10](https://github.com/user-attachments/assets/ceadfddb-b72a-4718-8982-09f2ae7cfe8c)
![2024-08-09_18 24 22](https://github.com/user-attachments/assets/3b651a2c-4afe-49bd-90b4-d151c589e727)

Very basic, trying to see if I can force the issue to appear. As it stands it happens seemingly at random and sometimes not at all during a gaming session.

**Comment by Gregie** [comment]
More info as well: I have the Lectern network hooked up to a Create Item Vault and an Occultism Dimensional Storage Actuator, both don't seem to suffer from the issue of not registering items.

**Comment by Gregie** [comment]
@shartte Yes, that chunk and all surrounding chunks loaded are force loaded by FTB Chunks.

**Comment by Gregie** [comment]
@shartte The Storage Bus sees all items _except_ the ones in the Storage Controller. So to answer your question it sees _some_ of the items accessible via that lectern.

**Comment by Gregie** [comment]
At least once cause seems to be adding drawers to the controller. You need to break and replace the Storage Bus for it to recognize that the items in the new drawer exist. Thinking back it may be THE cause for the bug to appear. I might've been messing with my storage system every time the issue occured.

**Comment by Gregie** [comment]
@shartte @NielsPilgaard So it seems like if you close the server after adjsting the storage drawers then the Storage Bus doesn't recognize any items in the Storage Controller. This is fixed by breaking and replaceing the Storage Bus before closing the server. (This would explain why it seemed random to me.) As you said maybe it's an Ars bug however, it does seem to only effect the Storage Bus. The Ars network is not hampered in functionality as far as I can tell. Good info for anyone else trying to do this kind of setup using these mods. Appreciate you looking into it!
